src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/User.orm.xml

```diff
@@ -49,10 +49,10 @@
             <cascade>
                 <cascade-all/>
             </cascade>
-            <join-column name="customer_id" referenced-column-name="id" nullable="false" />
+            <join-column name="customer_id" referenced-column-name="id" nullable="false" on-delete="CASCADE" />
```

Comment:
`on-delete="CASCADE"` will not help here, because it's softdeleteable issue. We will soon start getting rid of softdeleteable for all entities, so it will be resolved, but if you need it now you can add it to your project and remove the `<gedmo:soft-deleteable field-name="deletedAt" />` line.